packages/shared/src/graphql/users.ts

```diff
@@ -10,16 +10,6 @@ import { SourceMember } from './sources';
 import { graphqlUrl } from '../lib/config';
```

Comment:
Nice! Thank you for adding this cleanup ❤️ 

Just remembered: do we have the ticket to clean up the rest of legacy weekly rank? CSS files, etc. @nensidosari

Reply:
https://dailydotdev.atlassian.net/browse/MI-378

Reply:
I don't know why, but I think I have a personal vendetta against the legacy weekly goals, probably due to the problems it gave us back then 😂